src/pymoca/backends/casadi/model.py

```diff
@@ -108,12 +109,21 @@ def simplify(self, options):
 
             simple_parameters, symbols, values = [], [], []
             for p in self.parameters:
-                value = ca.MX(p.value)
-                if value.is_constant():
+                if isinstance(p.value, list):
```

Comment:
Why introduce this extra check?

```
In [6]: ca.MX([1,2,3]).is_constant()
Out[6]: True
```

Reply:
Doing that on higher order arrays will fail. E.g. `ca.MX(np.array([[1,2,3],[4,5,6]]))`

Reply:
Gotcha.